tests/testyamls/test-01.yaml

```diff
@@ -0,0 +1,45 @@
+group: test-1
```

Comment:
is there an assorted test for this? Note also #1916 should help with PRs like this